go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/budimanjojo/talhelper
+module github.com/budimanjojo/talhelper/v3
```

Comment:
shoot, missed this, sorry!

Reply:
I think this should work indeed

Reply:
Yeah this is the actual fix, the rest are just replacing everything to use this module :D